### PR TITLE
float8 readme: remove duplication

### PR DESCRIPTION
**Pull request by vkuzo**
We had two duplicate example training loops in float8 readme, removing and making the same example work for all recipes

### CODE REVIEW

**Review by danielvegamyhre — APPROVED** [review]

**Comment by danielvegamyhre** [comment]
Did we copy/paste + run the example and confirm it doesn't produce nans, loss goes down, etc? With learning rate of 0.1 on these toy models with a `y.sum().backward()` loss function, I've seen NaN losses. My theory is that since we are basically training the model to predict all 0s to minimize the sum, with the high learning rate of 0.1 it takes a large step into nan territory.

**Comment by vkuzo** [comment]
> Did we copy/paste + run the example and confirm it doesn't produce nans, loss goes down, etc?

Would be good for a future PR! It's just a toy example for now.
